frontend/src/App.vue

```diff
@@ -1,35 +1,35 @@
 <script>
+import 'normalize.css';
 import HeaderLogo from './components/HeaderLogo.vue';
+import { useUserStore } from './stores/user';
 
 export default {
   name: 'app',
+  setup() {
+    const userStore = useUserStore();
+    return { userStore };
+  },
   components: {
     HeaderLogo,
   },
 };
 </script>
 
 <template>
-  <nav>
-    <HeaderLogo />
-    <router-link class="link" to="/">Home</router-link>
-    <router-link class="link" to="/about">About</router-link>
+  <div>
+    <nav>
+      <HeaderLogo />
 
+      <div>
+        <router-link v-if="userStore.isLoggedIn" class="link" to="/">Home</router-link>
+        <router-link class="link" to="/about">About</router-link>
+        <router-link v-if="!userStore.isLoggedIn" class="link" to="/auth">Login</router-link>
```

Comment:
Nitpick: it is in general a better practice to use the `name` prop with `router-link` instead of the URL. Some of the reasons why can be found [here](https://router.vuejs.org/guide/essentials/named-routes.html).